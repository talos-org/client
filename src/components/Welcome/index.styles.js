// @flow
import styled from 'styled-components';
import { Box, Flex } from 'rebass';

export const ButtonGroup = styled(Box)`
  button {
    margin-top: 7px;
  }
`;

export const Container = styled(Flex)`
  min-height: 100vh;
`;
