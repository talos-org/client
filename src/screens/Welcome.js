// @flow
import * as React from 'react';

import WelcomeContainer from 'containers/WelcomeContainer';

export default () => <WelcomeContainer />;
