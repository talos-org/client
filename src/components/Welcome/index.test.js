import React from 'react';
import { shallow } from 'enzyme';

import Welcome from '.';

describe('Welcome', () => {
  it('should render correctly in “debug” mode', () => {
    const component = shallow(<Welcome debug />);

    expect(component).toMatchSnapshot();
  });
});
