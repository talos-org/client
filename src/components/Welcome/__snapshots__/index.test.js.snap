// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should render correctly in “debug” mode 1`] = `
<Route>
  <Component />
</Route>
`;
